kinto/core/permission/postgresql/__init__.py

```diff
@@ -363,15 +367,15 @@ def delete_object_permissions(self, *object_id_list):
         if len(object_id_list) == 0:
             return
 
-        object_ids_values = ','.join(["('^%s$')" % o.replace('*', '.*')
+        object_ids_values = ','.join(["('%s')" % o.replace('*', '%')
                                       for o in object_id_list])
         query = """
         WITH object_ids AS (
           VALUES %(object_ids_values)s
         )
         DELETE FROM access_control_entries
          USING object_ids
-         WHERE object_id ~ column1;"""
+         WHERE object_id LIKE column1;"""
```

Comment:
Note: according to docs (and performance obtained locally), LIKE behaves exactly like `=` if the values do not contain `%` character